src/main/java/com/stuypulse/robot/Robot.java

```diff
@@ -51,7 +52,9 @@ public void robotPeriodic() {
 
     /** This function is called once each time the robot enters Disabled mode. */
     @Override
-    public void disabledInit() {}
+    public void disabledInit() {
+        new ClimberBrakeCommand(robot.climber);
```

Comment:
this shouldn't be necessary. from blay: "we get disabled, and it fires automatically because it is a single solenoid" we do want the solenoid to be on(?) during the match the whole time. 

Think of the brake like a button. When it's held the brake is released. We want to hold it down when enabled and let go when disabled (and it lets go anyway when disabled)

Reply:
also it's new Command().schedule() -- all you're doing here is instantiating